### DISCUSSION TITLE
SlingerPlayer - does it work off LAN when remote accessing?

### DISCUSSION THREAD

**Discussion by tryin100** [GerryDazoo/SlingerPlayer#28] [Q&A] [answered]
Apologize up front so basic a question, but will SlingPlayer work to access remote slingboxes over the WAN or just when on the LAN?
thanks

**Comment by GerryDazoo** [comment, marked as answer]
It can work over the WAN to connect to a remote Slinger server.

**Reply by tryin100** [reply]
thanks
